docs/fields/file.md

```diff
@@ -32,5 +32,5 @@ field.
         attachment:
             type: file
             extensions: [ png, pdf ]
-            upload: portfolio
+        upload: portfolio
 ```
```

Comment:
Zero stress, I merged the PRs into one, and tweaked (the directory is now, and for a while, been created for you … that used to not be the case though) … and it is now merged into the repo with your name all over it :smile: 

Thanks mate, really appreciated :+1: